tuf_conformance/test_basic.py

```diff
@@ -4,7 +4,10 @@
 # system to build this eventually 
 
 import argparse
-from typing import Dict, List
+import json
+import os
+
+import pytest  # type: ignore
```

Comment:
what is this ignore for?

Reply:
unsure, copy-pasted from the sigstore conformance test setup